docs/features/README_ja.md

```diff
@@ -62,8 +62,8 @@ Networking レイヤーが含まれています。
 
 ## Ostracon Features
 
-* [Extending Tendermint-BFT with VRF-based Election](consensus_ja.md)
-* [BLS Signature Aggregation](signature_aggregation_ja.md)
+* [Extending Tendermint-BFT with VRF-based Election](consensus/index_ja.md)
+* [BLS Signature Aggregation](signature-aggregation/index_ja.md)
```

Comment:
Would you rename `index_ja.md` to `README.md`?

Reply:
ditto.